modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/types.ts

```diff
@@ -77,6 +74,15 @@ export interface ObjectDefinitionNodeData
 	objectFields: ObjectFieldNode[];
 }
 
+export type ObjectDefinitionNode = {
+	data: ObjectDefinitionNodeData;
+	id: string;
+	position: {
+		x: number;
+		y: number;
+	};
+	type: ObjectDefinitionNodeTypes;
+};
```

Comment:
Hey @igor-franca, you can remove this interface, this was replaced by the `Node` interface from React Flow

Reply:
Done!